lib/bot.js

```diff
@@ -251,6 +251,11 @@ class Bot {
       return;
     }
 
+    if (message.bot_id == 'B42KVL3PF') { // CHANGEME
```

Comment:
I was rather hoping to not have to configure it at all, by finding our own id at runtime. Didn't find a function to do that, but maybe I didn't look enough. Alternatively, we could make a note of the messages we send, and match them up with messages we get back.